### ISSUE TITLE
Installable package releases, NPM, brew, deb...

### ISSUE THREAD

**Issue by fool** [netlify/netlifyctl#26]
Since we will eventually complete the deprecation of netlify-cli, we'll need to provide netlifyctl via an npm wrapper of some sort.  

I'd say this is probably blocked on https://github.com/netlify/netlifyctl/issues/6 to give netlifyctl the same powers that netlify-cli has for site configuration instead of just deployment.

@rybit has suggested this is the eventual plan, not just a wish, but that there is no timeline for this update yet.

**Comment by calavera** [comment]
See the project roadmap board:  https://github.com/netlify/netlifyctl/projects/1

**Comment by calavera** [comment]
The more I think about this, the more I believe we should not make it available via NPM.

**Comment by calavera** [comment]
The more I think about this, the more I believe we should not make it available via NPM.

PROS:

- Installation for people that already have installed the previous CLI is the same.

CONS:

- NPM install is a hack.
- The only point in the PROS section is not really a PRO, it's only a NICE TO HAVE.
- NPM is not deterministic.
- NPM does not support the concept of package per ARCHITECTURE, which makes us hack the installation process.
- People are also familiar with their OS package manager, which is better at installing binary packages than NPM.
- Other package managers have much much better support for this, and we can get more help.

**Comment by fool** [comment]
ok, you need to fight with @biilmann on that, he declared it to be so and I passed it on to multiple customers @calavera .  I don't love it either - but what I really hate is that all of our docs and most of our customers use the current NPM package that we decided to stop maintaining 6 months ago and still haven't told them.

**Comment by fool** [comment]
I'm willing to call this a lost cause.  It's there in homebrew and available via curl these days, don't have a strong desire to see it in npm
